### ISSUE TITLE
Investigate custom watch mode for js bundling pipeline

### ISSUE THREAD

**Issue by lifeart** [embroider-build/embroider#1089]
ref: https://github.com/ember-cli/ember-cli/issues/9772

**Comment by ef4** [comment]
Yes, this is already something I'm planning to move toward. @krisselden has been laying the groundwork.

The summary is that rebuilds are faster if you let Webpack do its own file watching. It also does a better job automatically discovering and noticing changes to non-ember-addon dependencies that we would otherwise miss.
